### PR TITLE
[WIP] Add ui, db and actions for reminders

### PR DESCRIPTION
**Pull request by Procrat**
Continuing from #1.

This PR adds ui, db and actions for reminders.

Todo:
 - [x] Datepicker
 - [x] Add reminders to list
 - [x] Actually remind people with notifications
 - [x] Check how OSX notifications behave after reboot (spoilers: they persist)

### CODE REVIEW

**Comment by Procrat** [comment]
This PR is almost finished.
- Notifications work on Linux. @Silox: do you want to implement this for Mac OS in this PR?
- Currently, delayed notifications are handled by the system (with `at` on Linux). Should this at-job also handle the removal of the reminder-object in the database? If so, how? If not, how should we handle this? I guess this is again related to the larger question how Eva should integrate in ones workflow (e.g. as a stand-alone program or as a daemon?).

**Comment by Silox** [comment]
@Procrat: Yup, I'll implement it here :3

File: notifier.py [hunk 1]
```diff
@@ -12,10 +21,65 @@ def notify(message):
...
+        'Darwin': _delayed_notify_mac
+    }[platform.system()]
+    os_notify(reminder)
+
+
 def _notify_nix(message):
-    subprocess.check_call(['notify-send', '-a', APP_NAME, message])
+    try:
+        sp.check_call(['notify-send', '-a', APP_NAME, message])
+    except FileNotFoundError:
```
Comment (Silox) [inline notifier.py]: `FileNotFoundError` is not imported?

**Comment by Silox** [comment]
I placed a reminder and rebooted. It showed afterwards so I guess we can rely on the notificationcenter for the delayed notifications.

File: notifier.py (hunk 1 above)
Comment (Procrat) [inline notifier.py]: FileNotFoundError is a builtin Error ^^

**Comment by Procrat** [comment]
Naaiz

**Comment by Silox** [comment]
I guess this PR is completed?

**Comment by Procrat** [comment]
Looks like it :D

**Pull request merged**